config/api.yml

```diff
@@ -745,6 +745,12 @@
         :identifier: ems_cluster_protect
       - :name: unassign
         :identifier: ems_cluster_protect
+  :compliances:
+    :description: Compliances
+    :options:
+    - :subcollection
+    :verbs: *g
```

Comment:
what does the value `*g` mean?

Reply:
the verb set that can be used on the subcollection, in this case just the get

Reply:
higher-level that is like a "pointer" in YAML, and it means to insert the object referenced with the name "g".  g is defined here: https://github.com/ManageIQ/manageiq-api/blob/a62b5f3bb439ce50cc009908e76737d05d122651/config/api.yml#L14-L15